### ISSUE TITLE
[9.0] Fresh install database creation error

### ISSUE THREAD

**Issue by grazzianos** [odoo/odoo#9898]
Around October 15th (2015), on my test cenario on OSX (pulling all updates from Odoo Github), when I decided to erase a database and start another blank one, I saw the error below. So, to improve from where the error comes, I made a new install and cenario using Ubuntu 14.04.3 LTS with all requirements to Odoo and started to create a new database... same error again.

The curious fact is that, before this date (between the days 1 and 10 of october), this error did not exist. When I restored a previous backup database from this range of date, using the updated code from Github, everything run normally.

Seems like someone on Github changed a parameter name on the modules and the initial database creation becomes corrupt. How to solve?

PS: THE LANGUAGE OF DATABASE INSTALLATION IS **PORTUGUESE BRAZIL (pt_BR)**. 
**Another useful information: When I recovered a previous backup from a non problematic database, and reload translations (on administrative menu) with most updated code from Github, the error appears again. Problematic translation?**

ERROR
============================================================================
```python
2015-12-03 13:50:14,232 16825 INFO test_database openerp.tools.translate: loading /opt/odoo/addons/web_view_editor/i18n/pt_BR.po
2015-12-03 13:50:14,742 16825 INFO test_database werkzeug: 10.1.2.229 - - [03/Dec/2015 13:50:14] "POST /web/database/create HTTP/1.1" 303 -
2015-12-03 13:50:14,752 16825 INFO test_database openerp.addons.base.ir.ir_http: Generating routing map
2015-12-03 13:50:20,088 16825 INFO test_database werkzeug: 10.1.2.229 - - [03/Dec/2015 13:50:20] "GET /web/ HTTP/1.1" 500 -
2015-12-03 13:50:20,107 16825 ERROR test_database werkzeug: Error on request:
Traceback (most recent call last):
File "/usr/local/lib/python2.7/dist-packages/werkzeug/serving.py", line 177, in run_wsgi
execute(self.server.app)
File "/usr/local/lib/python2.7/dist-packages/werkzeug/serving.py", line 165, in execute
application_iter = app(environ, start_response)
File "/opt/odoo/openerp/service/server.py", line 245, in app
return self.app(e, s)
File "/opt/odoo/openerp/service/wsgi_server.py", line 184, in application
return application_unproxied(environ, start_response)
File "/opt/odoo/openerp/service/wsgi_server.py", line 170, in application_unproxied
result = handler(environ, start_response)
File "/opt/odoo/openerp/http.py", line 1422, in call
return self.dispatch(environ, start_response)
File "/opt/odoo/openerp/http.py", line 1396, in call
return self.app(environ, start_wrapped)
File "/usr/local/lib/python2.7/dist-packages/werkzeug/wsgi.py", line 588, in call
return self.app(environ, start_response)
File "/opt/odoo/openerp/http.py", line 1586, in dispatch
result = ir_http._dispatch()
File "/opt/odoo/addons/web_editor/models/ir_http.py", line 16, in _dispatch
return super(ir_http, self)._dispatch()
File "/opt/odoo/openerp/addons/base/ir/ir_http.py", line 176, in _dispatch
return self._handle_exception(e)
File "/opt/odoo/openerp/addons/base/ir/ir_http.py", line 147, in _handle_exception
return request._handle_exception(exception)
File "/opt/odoo/openerp/http.py", line 743, in _handle_exception
return super(HttpRequest, self)._handle_exception(exception)
File "/opt/odoo/openerp/addons/base/ir/ir_http.py", line 172, in _dispatch
result = request.dispatch()
File "/opt/odoo/openerp/http.py", line 774, in dispatch
r = self._call_function(*self.params)
File "/opt/odoo/openerp/http.py", line 316, in _call_function
return checked_call(self.db, *args, *kwargs)
File "/opt/odoo/openerp/service/model.py", line 118, in wrapper
return f(dbname, args, *kwargs)
File "/opt/odoo/openerp/http.py", line 312, in checked_call
result.flatten()
File "/opt/odoo/openerp/http.py", line 1373, in flatten
self.response.append(self.render())
File "/opt/odoo/openerp/http.py", line 1366, in render
context=request.context)
File "/opt/odoo/openerp/api.py", line 238, in wrapper
return old_api(self, args, *kwargs)
File "/opt/odoo/addons/web_editor/models/ir_ui_view.py", line 29, in render
return super(view, self).render(cr, uid, id_or_xml_id, values=values, engine=engine, context=context)
File "/opt/odoo/openerp/api.py", line 238, in wrapper
return old_api(self, args, *kwargs)
File "/opt/odoo/openerp/addons/base/ir/ir_ui_view.py", line 1065, in render
return self.pool[engine].render(cr, uid, id_or_xml_id, qcontext, loader=loader, context=context)
File "/opt/odoo/openerp/api.py", line 238, in wrapper
return old_api(self, args, *kwargs)
File "/opt/odoo/openerp/addons/base/ir/ir_qweb.py", line 254, in render
return self.render_node(element, qwebcontext, generated_attributes=qwebcontext.pop('generated_attributes', ''))
File "/opt/odoo/openerp/addons/base/ir/ir_qweb.py", line 297, in render_node
result = self.render_element(element, template_attributes, generated_attributes, qwebcontext)
File "/opt/odoo/openerp/addons/base/ir/ir_qweb.py", line 325, in render_element
raise_qweb_exception(message="Could not render element %r" % element.tag, node=element, template=template)
File "/opt/odoo/openerp/addons/base/ir/ir_qweb.py", line 320, in render_element
generated_attributes= name == "t" and generated_attributes or ''))
File "/opt/odoo/openerp/addons/base/ir/ir_qweb.py", line 295, in render_node
result = self._render_tagt_render
File "/opt/odoo/openerp/addons/base/ir/ir_qweb.py", line 459, in render_tag_call
res = self.render(cr, uid, template, d)
File "/opt/odoo/openerp/api.py", line 238, in wrapper
return old_api(self, args, *kwargs)
File "/opt/odoo/openerp/addons/base/ir/ir_qweb.py", line 252, in render
element = self.get_template(id_or_xml_id, qwebcontext)
File "/opt/odoo/openerp/addons/base/ir/ir_qweb.py", line 172, in get_template
document = qwebcontext.loader(name)
File "/opt/odoo/openerp/addons/base/ir/ir_ui_view.py", line 1063, in loader
return self.read_template(cr, uid, name, context=context)
File "/opt/odoo/openerp/api.py", line 238, in wrapper
return old_api(self, args, *kwargs)
File "/opt/odoo/openerp/addons/base/ir/ir_ui_view.py", line 938, in read_template
return self._read_template(cr, uid, view_id, context=context)
File "/opt/odoo/openerp/api.py", line 238, in wrapper
return old_api(self, args, *kwargs)
File "", line 2, in _read_template

File "/opt/odoo/openerp/tools/cache.py", line 85, in lookup
value = d[key] = self.method(args, *kwargs)
File "/opt/odoo/openerp/addons/base/ir/ir_ui_view.py", line 923, in _read_template
arch = self.read_combined(cr, uid, view_id, fields=['arch'], context=context)['arch']
File "/opt/odoo/openerp/api.py", line 238, in wrapper
return old_api(self, args, *kwargs)
File "/opt/odoo/openerp/addons/base/ir/ir_ui_view.py", line 635, in read_combined
view_arch = etree.fromstring(view['arch'].encode('utf-8'))
File "lxml.etree.pyx", line 3092, in lxml.etree.fromstring (src/lxml/lxml.etree.c:70473)

File "parser.pxi", line 1828, in lxml.etree._parseMemoryDocument (src/lxml/lxml.etree.c:106307)

File "parser.pxi", line 1716, in lxml.etree._parseDoc (src/lxml/lxml.etree.c:105098)

File "parser.pxi", line 1086, in lxml.etree._BaseParser._parseDoc (src/lxml/lxml.etree.c:99780)

File "parser.pxi", line 580, in lxml.etree._ParserContext._handleParseResultDoc (src/lxml/lxml.etree.c:94254)

File "parser.pxi", line 690, in lxml.etree._handleParseResult (src/lxml/lxml.etree.c:95690)

File "parser.pxi", line 620, in lxml.etree._raiseParseError (src/lxml/lxml.etree.c:94757)

QWebException: StartTag: invalid element name, line 1, column 17
2015-12-03 13:50:36,134 16825 INFO test_database openerp.addons.base.ir.ir_autovacuum: GC'd 0 user log entries
```
=========================================================================

**Comment by grazzianos** [comment]
I noticed, in table **ir.translation**, many occurrences of wrong translation with this terms:
```
&lt;!DOCTYPE html&gt; 

translated as 

<!DOCTYPE html>
```
When I corrected these lines, Odoo ran perfectly.

**Comment by rco-odoo** [comment]
@grazzianos there were some escaping issues in translations, indeed.
Those issues should have been fixed by b24d759036f7ebccd93df008fbfc2605efaeb574,
and @mart-e manually fixing translations coming from transifex.
I cannot reproduce the bug today (e5112ee00b1e552ab9127c910862accc4c9720c1).
